sdk/cosmosdb/Azure.ResourceManager.CosmosDB/tests/ScenarioTests/CassandraResourcesOperationsTests.cs

```diff
@@ -0,0 +1,170 @@
+
+            CassandraKeyspaceGetResults cassandraKeyspaceGetResults = cassandraKeyspaceResponse.Value;
+
+            Assert.NotNull(cassandraKeyspaceGetResults);
+            Assert.AreEqual(keyspaceName, cassandraKeyspaceGetResults.Name);
+
+            var cassandraKeyspaceResponse1 = await WaitForCompletionAsync(await cosmosDBClient.CassandraResources.StartCreateUpdateCassandraKeyspaceAsync(resourceGroupName, databaseAccountName, keyspaceName, cassandraKeyspaceCreateUpdateParameters));
+
+            CassandraKeyspaceGetResults cassandraKeyspaceGetResults1 = cassandraKeyspaceResponse1.Value;
+            Assert.NotNull(cassandraKeyspaceGetResults1);
+            Assert.AreEqual(keyspaceName, cassandraKeyspaceGetResults1.Name);
+
+            VerifyEqualCassandraDatabases(cassandraKeyspaceGetResults, cassandraKeyspaceGetResults1);
+
+            CassandraKeyspaceCreateUpdateParameters cassandraKeyspaceCreateUpdateParameters2 = new CassandraKeyspaceCreateUpdateParameters(new CassandraKeyspaceResource(keyspaceName2), new CreateUpdateOptions(sampleThroughput, default));
+
+            var cassandraKeyspaceResponse2 = await WaitForCompletionAsync(await cosmosDBClient.CassandraResources.StartCreateUpdateCassandraKeyspaceAsync(resourceGroupName, databaseAccountName, keyspaceName2, cassandraKeyspaceCreateUpdateParameters2));
+
+            CassandraKeyspaceGetResults cassandraKeyspaceGetResults2 = cassandraKeyspaceResponse2.Value;
+            Assert.NotNull(cassandraKeyspaceGetResults2);
+            Assert.AreEqual(keyspaceName2, cassandraKeyspaceGetResults2.Name);
+
+            var keyspacesResponse = cosmosDBClient.CassandraResources.ListCassandraKeyspacesAsync(resourceGroupName, databaseAccountName);
+            Task<List<CassandraKeyspaceGetResults>> taskCassandraKeyspaces = keyspacesResponse.ToEnumerableAsync();
+            List<CassandraKeyspaceGetResults> cassandraKeyspaces = taskCassandraKeyspaces.ConfigureAwait(false).GetAwaiter().GetResult();
+
+            Assert.NotNull(cassandraKeyspaces);
+
+            var throughputResponse = cosmosDBClient.CassandraResources.GetCassandraKeyspaceThroughputAsync(resourceGroupName, databaseAccountName, keyspaceName2);
+            ThroughputSettingsGetResults throughputSettingsGetResults = throughputResponse.ConfigureAwait(false).GetAwaiter().GetResult();
+            Assert.NotNull(throughputSettingsGetResults);
+            Assert.NotNull(throughputSettingsGetResults.Name);
+            Assert.AreEqual(throughputSettingsGetResults.Resource.Throughput, sampleThroughput);
+            Assert.AreEqual(cassandraThroughputType, throughputSettingsGetResults.Type);
+
+            //TODO: Migrate To Manual and Autoscale tests from example: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/cosmos-db/resource-manager/Microsoft.DocumentDB/stable/2020-04-01/examples/CosmosDBCassandraTableMigrateToAutoscale.json
```

Comment:
What is this?

Reply:
This was a test I found from the example json that wasn't covered in Track 1. The method to MigrateToManual didn't seem to work, so I've left a comment in case we ever want to cover this test. I can also remove the TODO: if that's preferred.

Reply:
What were the issues you were facing in MigrateToManual method ?

Reply:
I am getting a 404 error for "Entity not found". This is how I'm trying to use the method: 
`var migrateKeyspace = await CosmosDBManagementClient.CassandraResources.StartMigrateCassandraKeyspaceToManualThroughputAsync(resourceGroupName, databaseAccountName, keyspaceName);'. The resourceGroupName, databaseAccountName, and keyspaceName are valid before the call. Is there another way I should be using the method?